src/app/carbon-estimator-form/carbon-estimator-form.component.ts

```diff
@@ -187,6 +227,7 @@ export class CarbonEstimatorFormComponent implements OnInit {
 
   public resetForm() {
     this.estimatorForm.reset();
+    this.storeFormState();
```

Comment:
We could add some extra functionality to the storage service and clear the stored state here but I went with just storing it for now since it's slightly simpler.

Reply:
I decided to update this to clear the stored state.